build.gradle

```diff
@@ -17,14 +17,18 @@ configurations {
 
 repositories {
     mavenLocal()
-    maven { url "https://jitpack.io" }
+    maven { url "http://dl.bintray.com/4finance/jenkins-pipeline" }
     maven { url 'http://repo.jenkins-ci.org/releases/' }
+    maven { url "https://jitpack.io" }
     jcenter()
     mavenCentral()
 }
 
 dependencies {
-    lib 'io.fourfinanceit.pipeline:jenkins-pipeline-dsl:0.4.9'
+    
+    lib 'io.fourfinanceit.pipeline:jenkins-pipeline-core:0.5.0'
+    lib 'io.fourfinanceit.pipeline:jenkins-job-components:0.5.0'
+    lib 'io.fourfinanceit.pipeline:jenkins-pipeline-test-utils:0.5.0'
```

Comment:
Is it needed? Tests are executed earlier and job-dsl-plugin should be deal with them.